numpy/_core/defchararray.py

```diff
@@ -854,6 +892,12 @@ def isalnum(a):
     See Also
     --------
     str.isalnum
+
+    Examples
+    --------
+    >>> a = np.array(['a', '1', 'a1', ''])
```

Comment:
Also add an example with non-alphanumeric characters 
```suggestion
    >>> a = np.array(['a', '1', 'a1', '(', ''])
```

Reply:
Thank you for all your feedback. I will change this.